gratipay/utils/emails/queue_branch_email.py

```diff
@@ -7,6 +7,23 @@
 from gratipay import wireup
 
 
+def get_recently_active_participants(db):
+    return db.all("""
+        SELECT p.*::participants
```

Comment:
The double-colon is a [type cast](https://www.postgresql.org/docs/9.3/static/sql-expressions.html#SQL-SYNTAX-TYPE-CASTS). For the particular usage here, see the [postgres.py ORM docs](http://postgres-py.readthedocs.io/en/latest/#introducing-table-types).